ansible/roles/shared_data_dir/tasks/setup_host.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Create shared data dir
+  sudo: yes
+  file:
+    path: "{{ shared_data_dir }}"
+    owner: "nobody"
+    group: "{{ shared_dir_gid }}"
+    mode: "u=rwx,g=rwx,o=r"
+    state: directory
+  tags:
+    - shared_files
```

Comment:
Everything in this file was moved from `ansible/roles/shared_dir/tasks/setup_host.yml`. This line is the only thing that changed: `nfs` -> `shared_files`